src/irr/asset/CCPUMeshPacker.h

```diff
@@ -155,70 +165,123 @@ ReservedAllocationMeshBuffers CCPUMeshPacker<MDIStructType>::alloc(const Iterato
 }
 
 template <typename MDIStructType>
-void CCPUMeshPacker<MDIStructType>::createOutputBuffers()
+void CCPUMeshPacker<MDIStructType>::instantiateDataStorage()
 {
-	if (outputBuffer.indexBuffer.buffer != nullptr)
-		return;
-
 	//TODO: redo after safe_shrink fix
```

Comment:
you can use address_allocator_traits or the allocator itself and query the total size of its address space, then multiply that by the sizeof the units its working in and you'll get the buffer size you need.